__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const request = require("supertest");
+const app = require("../db/app");
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe("Get/api/topics", () => {
+  test("response with the status of 200 and all the available topics ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.map((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
```

Comment:
This is a good test but a map is not an appropriate array method for this. Can you think of a more appropriate array method that could be used instead?